Update StructMerge docs (#12540)
ydb-platform/ydb: ydb/docs/en/core/yql/reference/yql-core/builtins/_includes/struct.md

1. `left` - first structure.
2. `right` - second structure.

> 3. `mergeLambda` - _(optional)_ function to merge fields with the same name (arguments: field name, Optional field value of the first struct, Optional field value of the second struct - arguments are Nothing(T?) in case of absence of the corresponding struct field). By default, if present, the first structure's field value is used, and the second one's in other cases.

Reviewer: ```suggestion
3. `mergeLambda` - _(optional)_ function to merge fields with the same name (arguments: field name, `Optional` field value of the first struct, `Optional` field value of the second struct - arguments are `Nothing<T?>` in case of absence of the corresponding struct field). By default, if present, the first structure's field value is used; otherwise, the second one's value is used.
```
